Pipfile

```diff
@@ -5,6 +5,7 @@ verify_ssl = true
 
 [dev-packages]
 pre-commit = "~=1.13"
+mock = "~=3.0"
```

Comment:
Need to be under `[packages]`.

(Would like to re-consider which modules should be for dev-packages, packages)